CHANGELOG.md

```diff
@@ -4,6 +4,35 @@
 
 💥 Breaking changes:
 
+- Make radios and checkboxes components easier to link to from the error summary component
+
+  The first input in the radios and checkboxes will have an `id` and `name` attribute that match, allowing for easier generation of error summary links.
```

Comment:
This is only true if you haven't specified your own `idPrefix`, I think?

Reply:
I wonder if we should be more explicit about what's actually changed here?

Perhaps something like…

> The ID of the first input in a set of radios or checkboxes will no longer be suffixed with `-1`. This means that it will be the same as your `idPrefix`, or if `idPrefix` is not set it'll be the same as the `name`
>
> We've made this change to make it easier to link from the error summary to radios or checkboxes.